_episodes/identifiers.md

```diff
@@ -1,22 +1,85 @@
+
+If you think about it, these concepts are key to the success of large MIP projects in climatology, where many institutes and groups have made their data available to everyone.
+
+One of the challenges is how to deal with 'dynamic data', which is also highly relevant for climatological data. How do you make PIDs for data that is continuously evolving? [For the CMIP6 project](https://datascience.codata.org/articles/10.5334/dsj-2017-030/), a pragmatic approach was taken, consisting of a global PID for the entire, evolving data superset, and specific PIDs for citing specific subsets.
+
+> ## Discussion
+> Consider your own use case:
+> - Does the data have a DOI?
+> - Why/Why not?
+{: .challenge}
+
+How do you get a PID?
+---------------------
+
+Getting a PID (usually a DOI) for your data is as easy as publishing your data to a data archive. After the upload, a PID is automatically assigned. Note that just getting a PID is not all for making the data FAIR. Without metadata it will still be difficult to find the data. That's why many data archives make it easy to metadata, such as a list of authors, related projects, linked publications or other related items.
+
+Below you will find a list of resources to find a trusted or recommended repository.
+
+- [OpenAIRE](https://explore.openaire.eu/participate/deposit/learn-how):
+Find the appropriate repository to deposit your research products of any type (publication, data, software, other) or to include in your data management plan. Search and browse for OpenAIRE compliant repositories registered in OpenDOAR and re3data. OpenAIRE is a EC funded network of Open Access repositories, archives and journals that support Open Access policies.
+
+- [Zenodo](https://zenodo.org):
+Zenodo is a general-purpose open-access repository developed under the European OpenAIRE program and operated by CERN. It allows researchers to deposit data sets, research software, reports, and any other research related digital artifacts
+General purpose open-access repository developed
+
+- [FAIRsharing catalogue of databases](https://fairsharing.org/databases/):
+FAIRsharing is a web-based, searchable portal of three interlinked registries, containing both in-house and crowdsourced manually curated descriptions of standards, databases and data policies, combined with an integrated view across all three types of resource.
+
+- [Registry of Research Data Repositories (re3data.org)](https://www.re3data.org/):
+re3data.org is an Open Science tool that offers researchers, funding organizations, libraries and publishers an overview of existing international repositories for research data.
+
+- [European Research Council (ERC) Open Access](https://erc.europa.eu/funding-and-grants/managing-project/open-access):
+ERC curated webpage on policies on open access and data storage, including a list of recommended repositories.
+
```

Comment:
It might be nice to add another exercise/discussion here that ties in with the data access episode. There, I suppose we should have a discussion in which repo's are the most relevant. Subsequently, here, we can investigate how those repo's deal with DOIs.